package.json

```diff
@@ -71,6 +71,8 @@
     "classnames": "^2.2.5",
     "deepmerge": "^2.0.1",
     "dom-helpers": "^3.2.1",
+    "git-pull-request": "^0.3.2",
+    "global": "^4.3.2",
```

Comment:
Removed in https://github.com/mui-org/material-ui/commit/a97c6f179adef823351891c5200e816cd5f323af